CHANGELOG.md

```diff
@@ -1,13 +1,13 @@
 ## 2.6.0 / 2018-12-17
 
 * [CHANGE] Promtool: Remove the `update` command. #3839
+* [CHANGE] Remove default flags from the container's entrypoint. #4976
```

Comment:
It seems this text is not sufficient to describe the breakage that happened due to #4976. The new `/prometheus` symlink and storage location that are created in the Dockerfile are the main cause for the volume mount issues, right?